spec/models/user_spec.rb

```diff
@@ -1,13 +1,20 @@
 describe User do
+  let(:user) { create(:user) }
 
-  before(:each) { @user = FactoryBot.create(:user) }
-
-  subject { @user }
+  it 'should create a valid user with acceptable params' do
+    expect(user).to be_valid
+    expect(user).to be_a(User)
+  end
 
-  it { should respond_to(:name) }
+  it 'should not create a user with unacceptable params' do
+    @bad_user = user
```

Comment:
You should use shoulda matchers for these types of tests.

Reply:
added the gem. I'll re-write the tests to use shoulda-matchers